src/ExpressionEvaluator/Core/Source/ExpressionCompiler/DkmUtilities.cs

```diff
@@ -43,19 +43,25 @@ private static IEnumerable<DkmClrModuleInstance> GetModulesInAppDomain(this DkmC
                 });
         }
 
-        internal unsafe static ImmutableArray<MetadataBlock> GetMetadataBlocks(this DkmClrRuntimeInstance runtime, DkmClrAppDomain appDomain)
+        internal static ImmutableArray<MetadataBlock> GetMetadataBlocks(this DkmClrRuntimeInstance runtime, DkmClrAppDomain appDomain)
```

Comment:
Thank you! I hope I addressed this in the new iteration. Could you please take a look?

---
In reply to: [127111041](https://github.com/dotnet/roslyn/pull/20834#discussion_r127111041) [](ancestors = 127111041)